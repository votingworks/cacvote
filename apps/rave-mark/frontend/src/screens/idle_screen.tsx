import { H1, Main, P, Screen } from '@votingworks/ui';

export function IdleScreen(): JSX.Element {
  return (
    <Screen>
      <Main centerChild>
        <H1>Welcome</H1>
        <P>Insert your CAC to begin.</P>
      </Main>
    </Screen>
  );
}
