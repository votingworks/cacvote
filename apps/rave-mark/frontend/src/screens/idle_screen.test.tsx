import { renderWithThemes as render } from '@votingworks/ui';
import { IdleScreen } from './idle_screen';

test('renders', () => {
  const { container } = render(<IdleScreen />);
  expect(container).toMatchSnapshot();
});
