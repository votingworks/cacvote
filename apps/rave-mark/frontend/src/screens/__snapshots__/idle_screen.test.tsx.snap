// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    class="sc-dSCufp dqPMxK"
  >
    <main
      class="sc-fxwrCY hkigys"
    >
      <h1
        class="sc-dcJsrY fqlfQL"
      >
        Welcome
      </h1>
      <p
        class="sc-eqUAAy zDfVT"
      >
        Insert your CAC to begin.
      </p>
    </main>
  </div>
</div>
`;
